### ISSUE TITLE
Problem with shared contacts and permissions

### ISSUE THREAD

**Issue by reivaj01** [taers232c/GAMADV-X#13]
Hello,

I'm using shared contacts for a month aprox.

Today, an error ocurred: 

`Your client does not have permission to get URL <code>/m8/feeds/contacts/unknown/full</code> from this server.\n<ins>That&#39;s all we know.</ins></p></body></html>', 'reason': 'Cannot request contacts belonging to another domain (unknown`

I'm using the last version:

```
GAM 4.44.19 - https://github.com/taers232c/GAMADV-X
Ross Scroggs <ross.scroggs@gmail.com>
Python 2.7.13 64-bit final
google-api-python-client 1.6.1
Windows-10-10.0.14393 AMD64
Path: C:\GAM
Help: Syntax in file C:\GAM\GamCommands.txt
Help: Documentation is at https://github.com/taers232c/GAMADV-X/wiki
```

I think permissions are well:

  Scope: https://mail.google.com/                                    , Checked: PASS (1/12)
  Scope: https://sites.google.com/feeds                              , Checked: PASS (2/12)
  Scope: https://www.google.com/m8/feeds                             , Checked: PASS (3/12)
  Scope: https://www.googleapis.com/auth/activity                    , Checked: PASS (4/12)
  Scope: https://www.googleapis.com/auth/calendar                    , Checked: PASS (5/12)
  Scope: https://www.googleapis.com/auth/drive                       , Checked: PASS (6/12)
  Scope: https://www.googleapis.com/auth/gmail.settings.basic        , Checked: PASS (7/12)
  Scope: https://www.googleapis.com/auth/gmail.settings.sharing      , Checked: PASS (8/12)
  Scope: https://www.googleapis.com/auth/plus.login                  , Checked: PASS (9/12)
  Scope: https://www.googleapis.com/auth/plus.me                     , Checked: PASS (10/12)
  Scope: https://www.googleapis.com/auth/userinfo.email              , Checked: PASS (11/12)
  Scope: https://www.googleapis.com/auth/userinfo.profile            , Checked: PASS (12/12)

All scopes passed!
Service Account Client name 107326878622514630148 is fully authorized.

And, in the ManageOauthClients all is ok:

107326878622514630148
Email (Read/Write/Send)  https://mail.google.com/ 
https://sites.google.com/feeds 
https://www.google.com/m8/feeds 
https://www.google.com/m8/feeds/contacts/unknown/full 
https://www.googleapis.com/auth/activity 
https://www.googleapis.com/auth/admin 
Calendar (Read-Write)  https://www.googleapis.com/auth/calendar 
https://www.googleapis.com/auth/drive 
https://www.googleapis.com/auth/gmail.settings.basic 
https://www.googleapis.com/auth/gmail.settings.sharing 
https://www.googleapis.com/auth/plus.login 
https://www.googleapis.com/auth/plus.me 
https://www.googleapis.com/auth/script.external_request 
https://www.googleapis.com/auth/script.storage 
https://www.googleapis.com/auth/spreadsheets 
https://www.googleapis.com/auth/userinfo.email 
https://www.googleapis.com/auth/userinfo.profile 

As you can see, I added a new scope "https://www.google.com/m8/feeds/contacts/unknown/full" but nothing happens.

I searched Google and found nothing, and I don't know where I can find any clues.

Regards

**Comment by taers232c** [comment]
Delete the scope you added, it is not valid.
This: /m8/feeds/contacts/unknown/full should be /m8/feeds/contacts/domain.com/full
Google made a change that broke GAM, it is fixed in 4.44.19 but it may have affected you.

1) Recreate your oauth2.txt file
Do:  gam oauth create

2) Set customer_id/domain values in gam.cfg
Do: gam info domain
$ gam info domain
Customer ID: C12345678
Primary Domain: domain.com
...

Do: gam config verify
$ gam config verify
Section: DEFAULT
...
  customer_id = my_customer
...
  domain = ''
...

Do the values look like this or do they have the values from gam info domain?
If they look like this, do the following substituting your actual values:
gam config customer_id C12345678 domain domain.com save

Ross
--
ross.scroggs@gmail.com

**Comment by reivaj01** [comment]
Hello,

now, all works fine.

Thanks for all
